application/features/SFTP.py

```
@@ -136,9 +146,11 @@
     def chmod(self, path, mode, recursive):
         _, _, _, stderr = self.exec_command_blocking(
             f'chmod {"-R" if recursive else ""} {"{0:0{1}o}".format(mode, 3)} "{path}"')
+        logger.debug("SFTP: Change permission on " + path + " to '{0:0{1}o}'".format(mode, 3))
```

Ruff

Logging statement uses `+`

(G003)